### PR TITLE
Fix issue with retrieving FloatingIP

### PR DESCRIPTION
**Pull request by Jahsis**
Fix issue with retrieving FloatingIP data. Previously retrieved only object without populated data.

### CODE REVIEW

**Comment by haphan** [comment]
Can you post a reproducible code snippet? @Jahsis

File: src/Networking/v2/Extensions/Layer3/Models/FloatingIp.php [hunk 1]
```diff
@@ -70,6 +70,8 @@ public function delete()
 
     public function retrieve()
     {
-        $this->executeWithState($this->api->getFloatingIp());
+        $response = $this->execute($this->api->getFloatingIp(),
```
Comment (haphan) [inline src/Networking/v2/Extensions/Layer3/Models/FloatingIp.php]: Shouldn't this equivalent to the following?

```php
$response =  $this->executeWithState($this->api->getFloatingIp());
```

**Comment by haphan** [comment]
Good catch @Jahsis.
Please check my comment and let me know if its ok.

File: src/Networking/v2/Extensions/Layer3/Models/FloatingIp.php (hunk 1 above)
Comment (Jahsis) [reply src/Networking/v2/Extensions/Layer3/Models/FloatingIp.php]: Yes, looks like it is same. I guided myself more by other models in this project, for example:

https://github.com/php-opencloud/openstack/blob/master/src/Networking/v2/Models/Network.php#L53
https://github.com/php-opencloud/openstack/blob/master/src/Networking/v2/Models/Port.php#L149

And so on.

Comment (Jahsis) [reply src/Networking/v2/Extensions/Layer3/Models/FloatingIp.php]: Looks like your suggestion is much more cleaner, I will do changes.

**Comment by Jahsis** [comment]
@haphan please, check my changes.

**Pull request merged**